airflow/dags/ingestion/ask-astro-load-blogs.py

```diff
@@ -1,32 +1,42 @@
+import os
 from datetime import datetime
 
-import html2text
-import pandas as pd
-import requests
-from bs4 import BeautifulSoup
-from langchain.schema import Document
-from langchain.text_splitter import RecursiveCharacterTextSplitter
-from weaviate.util import generate_uuid5
-from weaviate_provider.hooks.weaviate import WeaviateHook
-from weaviate_provider.operators.weaviate import (
-    WeaviateCheckSchemaOperator,
-    WeaviateRetrieveAllOperator,
-)
+from include.tasks import extract, ingest, split
+from weaviate_provider.operators.weaviate import WeaviateCheckSchemaBranchOperator
 
 from airflow.decorators import dag, task
-from airflow.exceptions import AirflowException
 from airflow.providers.slack.operators.slack import SlackAPIPostOperator
 
-_WEAVIATE_CONN_ID = "weaviate_test"
-_SLACK_CONN_ID = "slack_api_default"
-
-default_args = {
-    "retries": 3,
-}
-
-
-# @dag(schedule_interval="0 5 * * *", start_date=datetime(2023, 9, 11), catchup=False, default_args=default_args)
-@dag(schedule_interval=None, start_date=datetime(2023, 9, 11), catchup=False, default_args=default_args)
+try:
+    ask_astro_env = os.environ["ASK_ASTRO_ENV"]
+except Exception:
+    ask_astro_env = "NOOP"
+
+if ask_astro_env == "prod":
+    _WEAVIATE_CONN_ID = "weaviate_prod"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "local":
+    _WEAVIATE_CONN_ID = "weaviate_local"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "dev":
+    _WEAVIATE_CONN_ID = "weaviate_dev"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "test":
+    _WEAVIATE_CONN_ID = "weaviate_test"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+else:
+    _WEAVIATE_CONN_ID = "weaviate_NOOP"
+    _SLACK_CONN_ID = "slack_api_NOOP"
```

Comment:
```suggestion
env = ["prod", "local", "test"]

_WEAVIATE_CONN_ID = "weaviate"
_SLACK_CONN_ID = "slack_api"
_GITHUB_CONN_ID = "github"

if ask_astro_env in env:
    _WEAVIATE_CONN_ID = f"{_WEAVIATE_CONN_ID}_{ask_astro_env}"
    _SLACK_CONN_ID = f"{_WEAVIATE_CONN_ID}_{ask_astro_env}"
    _GITHUB_CONN_ID = f"{_GITHUB_CONN_ID}_{ask_astro_env}"
else:
    _WEAVIATE_CONN_ID = f"{_WEAVIATE_CONN_ID}_NOOP"
    _SLACK_CONN_ID = f"{_WEAVIATE_CONN_ID}_NOOP"
    _GITHUB_CONN_ID = f"{_GITHUB_CONN_ID}_NOOP"
```
This looks a little bit redundant, what about?